### ISSUE TITLE
[Bug]: Cannot create superflat world

### ISSUE THREAD

**Issue by Tauryk** [FTBTeam/FTB-Modpack-Issues#813]
### Modpack

FTB Direwolf20 1.18

### Modpack version

1.2.0

### Describe the bug

unable to create a superflat world

### Steps to reproduce

launch the modpack
go to singleplayer
create new world
Set gamemode to creative (this should not have any impact but its what I did)
change world creation options to superflat
create new world

### Expected behaviour

I expected to create a superflat world for use in creative mode

### Additional information

One or more mods that modify the behaviour of minecraft terrain generation are preventing creation of superflat worlds.

**Comment by quietust** [comment]
Supposedly, this is caused by the "Terralith" mod, but I agree that it's definitely a bug.

**Comment by Sunekaer** [comment]
It's a bug we can't do anything about and it seems like a bug they won't fix sadly

**Comment by JacksonBailey** [comment]
From the Terralith Discord [#support](https://discord.com/channels/738046951236567162/869678422014697562) channel. Sounds like the workaround is to temporarily remove the Terralith mod and that this is a bug with Minecraft itself fixed in 1.19.

```
[11:10 AM] ???: just chatted this here to report a bug since there doesn't seem to be a bug report channel:
so everytime I try to create a Superflat world with Terralith on my folder it creates a Terralith world instead of Superflat world, I know I can temporary fix this by removing the mod itself from my mod folder but it would be convenient if I could create a superflat world without having to remove the mod from my folder
[11:12 AM] Tera: this is a datapack packaged as a mod, and that can't be applied to a worldgen datapack
[11:12 AM] ???: so just impossible
[11:12 AM] ???: ok
[...]
[1:07 PM] Blockheads: This is a bug in MC. It's fixed in 1.19.
```
